tests/test_api/test_api_facts.py

```diff
@@ -143,6 +146,39 @@ def test_get_fact_error_ignore(self):
             print_input=False,
             print_output=False,
             shell_executable='sh',
+            success_exit_codes=[0],
```

Comment:
Diff is messed up for some reason. I've however shamelessly copied the boilerplate from the existing test case.